taskcluster/fenix_taskgraph/__init__.py

```diff
@@ -36,6 +36,9 @@ def _import_modules(modules):
 
 
 def get_decision_parameters(graph_config, parameters):
+    parameters["tasks_for"] = "cron"
```

Comment:
TODO: These 2 lines are just for testing purpose and will be removed before landing the patch